modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/AssignQualityScore.pm

```diff
@@ -79,6 +80,11 @@ sub write_output {
     $self->dbc->disconnect_if_idle() if $self->dbc;
     $self->data_dbc->disconnect_if_idle();
 
+    my $member_type = $self->param('member_type');
```

Comment:
```suggestion
    my $member_type = $self->param_required('member_type');
```
Not sure about my suggestion, but it seems that `member_type` should be mandatory for the runnable to work properly in standalone.